scanner/enricher/fixedby/fixedby.go

```diff
@@ -90,16 +92,14 @@ func (e Enricher) Enrich(ctx context.Context, _ driver.EnrichmentGetter, vr *cla
 		fixedBy.Package = &p
 
 		// Set the Distribution.
-		// Just use the first one, as we only support single-distribution images.
+		// If we cannot not identify the distribution, then just use a dummy one,
+		// as we still want to support language-level packages.
+		fixedBy.Distribution = &claircore.Distribution{}
 		for _, d := range vr.Distributions {
+			// Just use the first one, as we only support single-distribution images.
 			fixedBy.Distribution = d
 			break
 		}
```

Comment:
This supposed to be in this PR? same change was also in https://github.com/stackrox/stackrox/pull/11337.

```suggestion
		// If we cannot identify the distribution, then use a dummy one,
		// as we still want to support language-level packages.
		fixedBy.Distribution = &claircore.Distribution{}
		for _, d := range vr.Distributions {
			// Just use the first one, as we only support single-distribution images.
			fixedBy.Distribution = d
			break
		}
```

Reply:
I plan on merging https://github.com/stackrox/stackrox/pull/11337 first anyway, so this will go away